pkg/virt-operator/util/config.go

```diff
@@ -20,56 +20,302 @@
 package util
 
 import (
+	"crypto/sha1"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
 	"os"
+	"reflect"
 	"regexp"
 	"strings"
 
 	k8sv1 "k8s.io/api/core/v1"
 
+	v1 "kubevirt.io/client-go/api/v1"
 	kvutil "kubevirt.io/kubevirt/pkg/util"
 )
 
 const (
 	// Name of env var containing the operator's image name
-	OperatorImageEnvName   = "OPERATOR_IMAGE"
+	OperatorImageEnvName        = "OPERATOR_IMAGE"
+	VirtApiShasumEnvName        = "VIRT_API_SHASUM"
+	VirtControllerShasumEnvName = "VIRT_CONTROLLER_SHASUM"
+	VirtHandlerShasumEnvName    = "VIRT_HANDLER_SHASUM"
+	VirtLauncherShasumEnvName   = "VIRT_LAUNCHER_SHASUM"
+	KubeVirtVersionEnvName      = "VERSION"
```

Comment:
We're introducing this var right? It might be better if it were a bit more KubeVirt specific? "VERSION" could conflict with other projects I'd imagine. Maybe "KUBEVIRT_VERSION"?